vllm/v1/serial_utils.py

```diff
@@ -174,7 +174,7 @@ def _encode_tensor(
     ) -> tuple[str, tuple[int, ...], Union[int, memoryview]]:
         assert self.aux_buffers is not None
         # view the tensor as a contiguous 1D array of bytes
-        arr = obj.flatten().contiguous().view(torch.uint8).numpy()
+        arr = obj.flatten().contiguous().cpu().view(torch.uint8).numpy()
```

Comment:
What is the reason for adding this?

I think this has the potential to introduce unnoticed performance regressions. It's probably better to require the tensors to already be on the CPU.